### PR TITLE
docs: use history_change option on SPA playgrounds 

### PR DESCRIPTION
**Pull request by lricoy**
## Changes

Just the docs, I started to do that long ago but there was some missing bits which this PR updates.

To recall, this change all SPA playgrounds to use `capture_pageview: 'history_change'` config to track `pageviews`.

If you can test this locally, that would be great. I've tried them all but it could have some edge cases. 

## Checklist
- [x] Tests for new code (see [advice on the tests we use](https://github.com/PostHog/posthog-js#tiers-of-testing))
- [x] Accounted for the impact of any changes across different browsers
- [x] Accounted for backwards compatibility of any changes (no breaking changes in posthog-js!)
- [x] Took care not to unnecessarily increase the bundle size

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -163,6 +163,9 @@ module.exports = {
             env: {
                 node: true,
             },
+            parserOptions: {
```
Comment (lricoy) [inline .eslintrc.js:166]: It seems playground was missing,

File: playground/error-tracking/nuxt-ts/plugins/posthog.client.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ export default defineNuxtPlugin((nuxtApp) => {
     const runtimeConfig = useRuntimeConfig()
     const posthogClient = posthog.init(runtimeConfig.public.posthogPublicKey, {
         api_host: runtimeConfig.public.posthogHost,
-        capture_pageview: false, // we add manual pageview capturing below
+        capture_pageview: 'history_change',
         capture_pageleave: true, // automatically capture a pageleave event when the user leaves the site or closes the tab
```
Comment (robbie-c) [inline playground/error-tracking/nuxt-ts/plugins/posthog.client.js:9]:
```suggestion
```
We can delete this line too, as the default is `if_capture_pageview`

File: playground/nuxtjs/plugins/posthog.client.js [hunk 3]
```diff
@@ -1,28 +1,18 @@
...
+import { defineNuxtPlugin, useRuntimeConfig } from '#imports'
 
 import posthog from 'posthog-js'
 export default defineNuxtPlugin(() => {
     const runtimeConfig = useRuntimeConfig()
     const posthogClient = posthog.init(runtimeConfig.public.posthogPublicKey, {
         api_host: runtimeConfig.public.posthogHost,
-        capture_pageview: false, // we add manual pageview capturing below
+        capture_pageview: 'history_change',
         capture_pageleave: true, // automatically capture a pageleave event when the user leaves the site or closes the tab
```
Comment (robbie-c) [inline playground/nuxtjs/plugins/posthog.client.js:10]:
```suggestion
```

**Review by robbie-c — APPROVED** [review]
LGTM with 2 tiny changes

**Comment by mirshko** [comment]
@robbie-c is this safe to use in production to replace the currently suggested PageView useEffect and manual $pageview captures?

**Comment by robbie-c** [comment]
@mirshko yes!
